provision.bash

```diff
@@ -2,24 +2,37 @@
 
 set -e  # Exit script if any error
 
+# The shared folder is specified in Vagrantfile.
+VAGRANT_SHARED_FOLDER=/home/vagrant/desktop
+
+. $VAGRANT_SHARED_FOLDER/sqlflow/docker/dev/find_fastest_resources.sh
```

Comment:
Can remove the prefix `.`.

Reply:
here, we use . to import the function from `find_fastest_resources.sh` according to https://stackoverflow.com/questions/9772036/pass-all-variables-from-one-shell-script-to-another